packages/xo-web/src/xo-app/home/index.js

```diff
@@ -478,6 +486,7 @@ export default class Home extends Component {
   }
 
   state = {
+    vmsFilter: DEFAULT_TYPE === 'VM' ? VM_FILTERS[0] : {},
```

Comment:
` DEFAULT_TYPE === 'VM'` will be 100% `true`. 

This condition isn't needed because the `vmsFilter` will only be used in case of VMs.